targets/ubuntu-14.04-enterprise/Dockerfile

```diff
@@ -115,3 +97,7 @@ RUN echo $IMAGE_TAG > /opt/circleci/image_version
 ADD targets/ubuntu-14.04-enterprise/pkg-versions.sh /opt/circleci/bin/pkg-versions.sh
 
 LABEL circleci.user="ubuntu"
+
+USER ubuntu
+
+CMD ["/bin/bash"]
```

Comment:
I see. The XL image is for picard only where you don't need `sbin/init` but it's required in 1.0. I wasn't fully sure if adding `CMD` breaks but sounds like it did as George mentioned.